github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.32.yaml

```diff
@@ -0,0 +1,34 @@
+---
+presubmits:
+  k8snetworkplumbingwg/ovs-cni:
+    - name: pull-e2e-ovs-cni-0.32
+      context: pull-e2e-ovs-cni
+      branches:
+        - release-0.32
+      always_run: true
+      optional: false
+      decorate: true
+      decoration_config:
+        timeout: 3h
+        grace_period: 5m
+      max_concurrency: 6
+      labels:
+        preset-podman-in-container-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      cluster: prow-workloads
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20230801-94954c0
```

Comment:
are you fine with that old image?
asking because now i have other problems that might be related to image (not sure yet)
i guess you just took it based on the one version above / below and then it is fine as long as it works

Reply:
I'd keep it in sync with  other stable branches. If that turns out to be problematic, I'd bump all of them at once